core/pom.xml

```diff
@@ -161,6 +161,13 @@
             <scope>provided</scope>
         </dependency>
 
+        <!-- Constructing objects without using constructors -->
+        <dependency>
+            <groupId>org.objenesis</groupId>
+            <artifactId>objenesis</artifactId>
+            <version>2.6</version>
```

Comment:
Can you add this to the dependancyManagement section of the bom
- https://github.com/DozerMapper/dozer/blob/master/bom-dependencies/pom.xml